live-examples/js-examples/arraybuffer-slice.html

```diff
@@ -0,0 +1,21 @@
+<pre>
+<code id="static-js">// create an ArrayBuffer with a size in bytes
+  var buffer = new ArrayBuffer(16);
+
+  var int32View = new Int32Array(buffer); // create the view
+
+  for (var i = 0; i < int32View.length; i++) {
+    int32View[i] = i;
+  }
```

Comment:
I don't have any bright ideas here really, but is there a more concise/simple way we can initialise this array? It feels to me like this is quite extraneous to the example...